Tests/BSONTests/BSONCorpusTests.swift

```diff
@@ -129,97 +156,101 @@ final class BSONCorpusTests: BSONTestCase {
+                        expect(try canonicalEncoder.encode(BSONDocument(fromBSON: dBData)))
+                            .to(cleanEqual(test.canonicalExtJSON), description: test.description)
 
-                        expect(decimal128FromString.description).to(equal(decimal128CorpusString))
-                        expect(decimal128FromBinary.description).to(equal(decimal128CorpusString))
-                    }
+                        // bson_to_relaxed_extended_json(dB) = rEJ (if rEJ exists)
+                        if let rEJ = test.relaxedExtJSON {
+                            expect(try relaxedEncoder.encode(dBData)).to(cleanEqual(rEJ), description: test.description)
+                        }
 
-                    // native_to_canonical_extended_json( bson_to_native(cB) ) = cEJ
-                    // expect(docFromCB.canonicalExtendedJSON).to(cleanEqual(test.canonicalExtJSON))
+                        let docFromDB = try BSONDocument(fromBSON: dBData)
 
-                    // native_to_relaxed_extended_json( bson_to_native(cB) ) = rEJ (if rEJ exists)
-                    // if let rEJ = test.relaxedExtJSON {
-                    //     expect(try Document(fromBSON: cBData).extendedJSON).to(cleanEqual(rEJ))
-                    // }
+                        // native_to_bson( bson_to_native(dB) ) = cB
+                        expect(docFromDB.toData()).to(equal(cBData), description: test.description)
 
-                    // for cEJ input:
-                    // native_to_canonical_extended_json( json_to_native(cEJ) ) = cEJ
-                    // expect(try Document(fromJSON: cEJData).canonicalExtendedJSON)
-                    //        .to(cleanEqual(test.canonicalExtJSON))
-
-                    // // native_to_bson( json_to_native(cEJ) ) = cB (unless lossy)
-                    // if !lossy {
-                    //     expect(try Document(fromJSON: cEJData).rawBSON).to(equal(cBData))
-                    // }
-
-                    // for dB input (if it exists):
-                    // if let dB = test.degenerateBSON {
-                    //     guard let dBData = Data(hexString: dB) else {
-                    //         XCTFail("Unable to interpret degenerate_bson as Data")
-                    //         return
-                    //     }
-
-                    //     // bson_to_canonical_extended_json(dB) = cEJ
-                    //     expect(try Document(fromBSON: dBData).canonicalExtendedJSON)
-                    //         .to(cleanEqual(test.canonicalExtJSON))
-
-                    //     // bson_to_relaxed_extended_json(dB) = rEJ (if rEJ exists)
-                    //     if let rEJ = test.relaxedExtJSON {
-                    //         expect(try Document(fromBSON: dBData).extendedJSON).to(cleanEqual(rEJ))
-                    //     }
-                    // }
+                        // native_to_canonical_extended_json( bson_to_native(dB) ) = cEJ
```

Comment:
Can we also add a comment before this one and the next one indicating that they're not in the spec yet but may be in the future, including the ticket number DRIVERS-1355?